packages/app/src/components/ConnectionForm.tsx

```diff
@@ -134,9 +136,15 @@ export function ConnectionForm({
   const deleteConnection = useDeleteConnection();
 
   const onSubmit = (data: Connection) => {
+    // Make sure we don't save a trailing slash in the host
+    const normalizedData = {
+      ...data,
+      host: stripTrailingSlash(data.host),
```

Comment:
Probably out of scope. I wonder if we should handle this on the API side to always remove the trailing slash before any update/insert operations since I suspect the clickhouse client would throw with the malformed host

Reply:
We definitely should also be handling this in the API, but it was a slippery slope exclusively making that solution. I'll add in the API side as well.